content/docs/security/cors.md

```diff
@@ -21,28 +25,35 @@ CORS policies are typically implemented to limit access to server resources for
 * A JavaScript on a web page at `example.com` tries to access a different port, such as `example.com:3001`.
 * A JavaScript on a web page at `https://example.com` tries to access the resources by using a different protocol, such as `http://example.com`.
 
-{{% callout type="info" %}} 
-Some apps, such as `httpbin`, have built-in CORS policies that allow all origins. These policies take precedence over CORS policies that you might configure in kgateway. 
-{{% /callout %}}
+### Configuration options {#options}
+
+You can configure the CORS policy at two levels:
+
+* HTTPRoute: For the native way in Kubernetes Gateway API, configure a CORS policy in the HTTPRoute. The policy is applied to all the routes that are defined in the HTTPRoute.This route-level policy takes precedence over any TrafficPolicy CORS that you might configure. For more information, see the [Kubernetes Gateway API docs](https://gateway-api.sigs.k8s.io/reference/spec/#httpcorsfilter) and [CORS design docs](https://gateway-api.sigs.k8s.io/geps/gep-1767/).
```

Comment:
Just a minor correction that the `HTTPRouteFilter` (which has the `HTTPCORSFilter`) can also [appear in BackendRef](https://gateway-api.sigs.k8s.io/reference/spec/#httproutefilter).
It basically means that if you have multiple backendrefs you can assign CORS to only selection of them. Most of the cases there is only one but worth mentioning the flexibility.
This is also something that can't be configured in `TrafficPolicy` but can in `HTTPRoute` so they kind of complete each other to cover all levels.